core/iwasm/interpreter/wasm_loader.c

```diff
@@ -2033,6 +2033,12 @@ load_type_section(const uint8 *buf, const uint8 *buf_end, WASMModule *module,
             if (flag == DEFINED_TYPE_REC) {
                 read_leb_uint32(p, p_end, rec_count);
 
+                if (rec_count == 0) {
+                    set_error_buf(
+                        error_buf, error_buf_size,
+                        "recursive type count should be greater than 0");
```

Comment:
Not sure about that `rec_count` can't be 0. Would you mind pointing out the relevant content in the GC spec? https://webassembly.github.io/gc/core/valid/modules.html